src/components/DeleteSwitch.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <q-btn id="delete-switch"
+         color="negative"
```

Comment:
I feel like making the button not be red when the delete mode is off, would make it more apparent when the user is in delete mode.

Reply:
This is true, I added an alternative color when the delete mode is off.